addon/components/bootstrap-selectize.js

```diff
@@ -0,0 +1,55 @@
+import Ember from 'ember';
+import layout from '../templates/components/bootstrap-selectize';
+import InputableMixin from '../mixins/component/inputable';
+
+export default Ember.Component.extend(InputableMixin, {
+  classNames: ['form-group', 'bootstrap-selectize-component']
+  isSrOnly: false,
+  selectOnTab: true,
+  classNameBindings: ['hasSuccess:has-success', 'hasWarning:has-warning', 'hasError:has-error '],
+  plugins: 'remove_button',
+  labelColumns: 2,
+  inputColumns: 10,
+
+  classNames: ['bootstrap-selectize-component', 'form-group'],
+
+  createItem: computedActionKey('create-item'),
```

Comment:
Looks like you need to throw the computedActionKey decorator into a util file and import it.